Dockerfile

```diff
@@ -1,7 +1,22 @@
-FROM ruby:2.5.1
-
-RUN apt-get update
-RUN apt-get install -y build-essential libpq-dev mysql-client && rm -rf /var/lib/apt/lists/*
+FROM ruby:2.5.1-alpine
+RUN apk update && \
+    apk add --no-cache \
+      alpine-sdk \
+      libstdc++ \
```

Comment:
Could you please sort this list of libraries?

Reply:
Fixed. Please check again.